docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -392,85 +497,62 @@ Accept: application/json
 }
 ```
 
-If no shipping quotes are available, the shipping carrier will send a response with the following format for the shipping quote:
+<!-- type: tab-end  -->
+
+If no shipping quotes are available, the your carrier will send a response with the following format for the shipping quote:
 
-```json title="Example response: No shipping rates" lineNumbers
+```json title="Example POST response" lineNumbers
 {
   "quote_id": "example_quote",
   "messages": [],
   "carrier_quotes": []
 }
 ```
 
-When you uninstall an app with an associated shipping carrier, you also automatically remove all the shipping methods and connection info for that carrier from the store. You can no longer make quote requests, and users will no longer see shipping quotes for that carrier.
-
 <!-- theme: info -->
 > #### Note
-> The response payload will display shipping quotes from lowest to highest price.
-
-## Including product metadata in rate requests
-
-BigCommerce passes carrier-specific product metadata to a carrier service in a rate request via product and variant metafields. This product metadata can be useful if your service depends on specific fields that are not existent on BigCommerce products or variants by default.
-
-To pass metadata in a rate request, the metafield must meet the following requirements:
-
-- it must be a product or variant metafield (you cannot pass category, brand, or other metafields in rate requests)
-- the metafield `permission_set` must be `read` or `write`
-- the metafield `namespace` must match this format: `shipping_carrier_<carrier_id>` (example: `shipping_carrier_72`)
-
-
-The carrier registration process described in the [Sign up](#sign-up) section provides the `carrier_id`.
-
-For more information on product and variant metafields, see:
-
-- [API Reference > Store Management > Catalog > Product Metafields](/api-reference/store-management/catalog/product-metafields)
-- [API Reference > Store Management > Catalog > Product Variant Metafields](/api-reference/store-management/catalog/product-variants-metafields)
-
-## Submitting the app
-
-Before submitting your app, make sure you have the following information.
-
-**Single-carrier or multi-carrier**
-
-A single-carrier app will offer one service, such as USPS. A multi-carrier app will offer more than one carrier such as USPS, DHL, and Canada Post.
+> The response displays shipping quotes from lowest to highest price.

```

Comment:
Does BC doe this, or does the shipping provider app? I _think_ BC will do this automatically regardless of the order in which the shipping provider responds.

Reply:
I have clarified the statement to say that BC orders the responses